aya/src/bpf.rs

```diff
@@ -856,6 +861,29 @@ impl Bpf {
         self.maps.iter().map(|(name, map)| (name.as_str(), map))
     }
 
+    /// An iterator over all the maps that allows modification.
```

Comment:
A mutable iterator over all the maps.

Reply:
Done! I think this is what I had before all the reviews 😜